packages/notifier/exported.d.ts

```diff
@@ -0,0 +1,43 @@
+/** @file Declare common ambient types */
+/* eslint-disable import/named */
+import {
+  EachTopic as _EachTopic,
```

Comment:
somewhere I saw a good explanation of why these gymnastics are used, including a stackoverflow reference; is it convenient to add a pointer from here?

Reply:
I resisted coping it an Nth time. I'll write up some docs in Markdown and have the modules reference it